ERROR:api:Request to http://localhost:8000/post/None failed: 400 Client Error: Bad Request for url: http://localhost:8000/post/None
ERROR:api:Request to http://localhost:8000/post/None failed: 400 Client Error: Bad Request for url: http://localhost:8000/post/None
ERROR:api:Request to http://localhost:8000/post/None failed: 400 Client Error: Bad Request for url: http://localhost:8000/post/None
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c219450>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e620>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e7a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c219060>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21a0b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting3 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21a410>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21ad70>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21a6e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c218dc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c219c00>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c23c880>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c23f370>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c23e6b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e860>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c23e110>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c23cd00>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21bbb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21b310>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c23e5f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c23fc10>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f2b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f550>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35d120>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21a140>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35c340>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35fe80>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e680>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e7d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f8b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f910>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e920>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35c7c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting3 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e2f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35c040>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35fe50>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21b9d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c21b400>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35fcd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f5b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting3 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35c8b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d8bb08d90>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e7a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35fb50>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35ead0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f310>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35ecb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting3 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35c070>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d89598eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f820>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35fe50>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35ef80>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35c970>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35f220>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting3 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c313a90>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35ee30>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e2f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35d240>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/subreddit-flairs/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /subreddit-flairs/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d895a4ca0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/post/tylerdurdentesting1 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /post/tylerdurdentesting1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d8959a830>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/post/tylerdurdentesting2 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /post/tylerdurdentesting2 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35c6a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/post/tylerdurdentesting3 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /post/tylerdurdentesting3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35ea40>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/post/tylerdurdentesting4 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /post/tylerdurdentesting4 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35e950>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/post/tylerdurdentesting5 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /post/tylerdurdentesting5 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c218190>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/post/tylerdurdentesting6 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /post/tylerdurdentesting6 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c313f10>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:api:Request to http://localhost:8000/post/tylerdurdentesting7 failed: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /post/tylerdurdentesting7 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x744d5c35d5d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
